pcsx2/MemoryCardFile.cpp

```diff
@@ -371,7 +371,11 @@ void FileMemoryCard::Open()
 			m_chkaddr = 0x210;
 
 			if (!m_ispsx[slot] && FileSystem::FSeek64(m_file[slot], m_chkaddr, SEEK_SET) == 0)
-				std::fread(&m_chksum[slot], sizeof(m_chksum[slot]), 1, m_file[slot]);
+			{
+				const size_t read_result = std::fread(&m_chksum[slot], sizeof(m_chksum[slot]), 1, m_file[slot]);
+				if (read_result == 0)
+					Console.Error("(FileMcd) error reading memcard.");
```

Comment:
Maybe it's advisable to use Host::ReportFormattedErrorAsync() so the failure to read doesn't go unnoticed by the user?

Reply:
@RedPanda4552 thoughts?

Reply:
Agreed, I would use the function referenced by fobes (same one as just up in the main if block).